Gemfile

```diff
@@ -64,3 +64,4 @@ gem "sitemap_generator", "~> 6.1"
 gem "stripe", ">= 2.8", "< 6.0"
 gem "timezone_finder", "~> 1.5.7"
 gem "tzinfo-data", platforms: [:mingw, :mswin, :x64_mingw, :jruby]
+gem "validate_url"
```

Comment:
I'm not sold on adding this dependency here since we aren't doing it for the developer's website. What's your reasoning for needing to validate this?

Reply:
I was on the fence about it as well, I will remove it and just use a regex to validate the field

Reply:
It looks like we can wrap [`Addressable::URI`](https://www.rubydoc.info/gems/addressable/Addressable/URI) in a validator and that would get us pretty far.